src/TwigComponent/src/DependencyInjection/TwigComponentExtension.php

```diff
@@ -110,10 +110,7 @@ class_exists(AbstractArgument::class) ? new AbstractArgument(\sprintf('Added in
         $container->register('.ux.twig_component.twig.component_runtime', ComponentRuntime::class)
             ->setArguments([
                 new Reference('ux.twig_component.component_renderer'),
-                service_locator([
-                    'ux:icon' => service('.ux_icons.twig_icon_runtime')->nullOnInvalid(),
-                    'ux:map' => service('ux_map.twig_runtime')->nullOnInvalid(),
-                ]),
+                new ServiceLocatorArgument(new TaggedIteratorArgument('.ux.twig_component.twig_renderer')),
             ])
             ->addTag('twig.runtime')
```

Comment:
BTW, all these service declarations might better live in a services.php config file to me. For another PR :)

Reply:
Hah, agreed! This comes from a time when the component was much simpler and Ryan and I were just hacking on it.